src/styles/blocks/calendar.scss

```
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  max-width: $calendar-width;
+  gap: $gap-width;
+  padding: 40px $calendar-side-paddings 0;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+    @include square($day-size);
+    border: 1px solid #000;
+    background-color: #eee;
+    transition: ease 0.5s;
+    cursor: pointer;
+
+    &:hover {
+      background-color: #ffc0cb;
+      transform: translateY(-20px);
+    }
+
+    &::before {
+      position: absolute;
+    }
+
+    @for $index from 1 through 31 {
+      &:nth-child(#{$index})::before {
+        content: "#{$index}";
+      }
+    }
+  }
+
+  @each $day, $number in $days-count {
+    &--start-day--#{$day} {
+      & :first-child {
```

You can combine these selectors
```suggestion
    &--start-day--#{$day} :first-child {
```